src/components/Search/CompareWithBase.tsx

```diff
@@ -87,6 +87,7 @@ function CompareWithBase({ mode, view }: CompareWithBaseProps) {
           <div className='compare-card-tagline'>{strings.base.tagline}</div>
         </div>
         <div
+          role='img'
```

Comment:
I think i'd use `aria-hidden="true"` for that one, it doesn't provide anything meaningful to the page, as the text besides  already has everything needed.